### PR TITLE
MER-1056 return values from function, innerText iso innerHtml for sanatizing

### PR DESCRIPTION
**Pull request by peterDijk**

### CODE REVIEW

File: textFit.js [hunk 1]
```diff
@@ -64,99 +66,133 @@
...
+      elType !== "[object HTMLCollection]"
+    ) {
       els = [els];
     }
 
+    const items = [];
     // Process each el we've passed.
-    for(var i = 0; i < els.length; i++){
-      processItem(els[i], settings);
+    for (var i = 0; i < els.length; i++) {
```
Comment (insitus) [inline textFit.js]:
```suggestion
    const elsLength = els.length;
    for (let i = 0; i < elsLength; i++) {
```

Comment (insitus) [reply textFit.js]: let instead of var and a small performance improvement

File: textFit.js [hunk 2]
```diff
@@ -64,99 +66,133 @@
 
     // Support passing a single el
     var elType = Object.prototype.toString.call(els);
-    if (elType !== '[object Array]' && elType !== '[object NodeList]' &&
-            elType !== '[object HTMLCollection]'){
+    if (
+      elType !== "[object Array]" &&
+      elType !== "[object NodeList]" &&
+      elType !== "[object HTMLCollection]"
```
Comment (Kageetai) [inline textFit.js]: I am not really sure wether this is what you are intending here, but wouldn't it be better to use `typeof` here?

File: textFit.js [hunk 3]
```diff
@@ -64,99 +66,133 @@
...
+      }
     }
 
     // Add textFitted span inside this container.
-    if (originalHTML.indexOf('textFitted') === -1) {
-      innerSpan = document.createElement('span');
-      innerSpan.className = 'textFitted';
+    if (originalHTML.indexOf("textFitted") === -1) {
+      innerSpan = document.createElement("span");
+      innerSpan.className = "textFitted";
```
Comment (insitus) [inline textFit.js]: Why are you using double quotes?

File: textFit.js [hunk 4]
```diff
@@ -64,99 +66,133 @@
...
     }
 
+    const items = [];
     // Process each el we've passed.
-    for(var i = 0; i < els.length; i++){
-      processItem(els[i], settings);
+    for (var i = 0; i < els.length; i++) {
+      items.push(processItem(els[i], settings));
     }
+    return items;
```
Comment (Kageetai) [inline textFit.js]: Isn't this whole thing not just a map?
```
return els.map(el => processItem(el, settings);
```

File: textFit.js (hunk 3 above)
Comment (peterDijk) [reply textFit.js]: i think prettier changed that

File: textFit.js (hunk 4 above)
Comment (peterDijk) [reply textFit.js]: true. I haven't changed anything apart from the stuff we needed. I'll go through the code to do that

**Comment by peterDijk** [comment]
> It looks like a lot of changes here are just prettier or some other code styling, no? But I don't see any definition for that anywhere in this package.

It's my editor that does that

**Pull request merged**